src/RG.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+//------------------------------------------------------------------------------
+// code development settings
+//------------------------------------------------------------------------------
+
+// to turn on Debug for a single file add:
+// #define RG_DEBUG
+// just before the statement:
+// #include "RG.h"
```

Comment:
Does this properly scope `RG_DEBUG` to that file without a corresponding `undef`?

Reply:
If you define `RG_DEBUG` before including `RG.h` in some **source** file, that file only will have `ASSERT` defined to do actual work.